src/app/lib/utilities/getEmbedUrl/index.test.js

```diff
@@ -147,6 +148,29 @@ const testCases = [
       queryString: liveOverrideParam,
     },
   },
+  {
```

Comment:
Could we also have tests without the override - see scenarios at top of file

Reply:
Added